scripts/downloader.py

```diff
@@ -0,0 +1,77 @@
+import asyncio
+import os
+
+from requests import get
+
+
+DOWNLOAD_DIR = './_downloaded'
```

Comment:
Šlo by to prosím nastavit jako podsložku v project rootu a tu složku dát do `.gitignore`? Přes `pathlib` by to nemělo být moc složité:

```python
DOWNLOAD_DIR = Path(__file__).parent.parent / 'backups'
```

Název složky by se mi líbil "backups".